.github/ISSUE_TEMPLATE/suggest-a-new-rule.md

```diff
@@ -0,0 +1,20 @@
+---
+name: New rule suggestion
+about: Suggest a new rule
+title: '[New rule] '
+labels: ''
+assignees: ''
+
+---
+
+## Describe the problem you are trying to solve
+
+<!-- A clear and concise description of what you want to the rule to prevent. -->
+
+## Describe suggestions on how to achieve the rule
+
+<!-- A clear description to how to achieve the rule. -->
```

Comment:
💡 It would be good to also describe how code should be automatically fixed when the issue occurs.